src/include/catalog/config_catalog.h

```diff
@@ -0,0 +1,76 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// config_catalog.h
+//
+// Identification: src/include/catalog/config_catalog.h
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+
+#pragma once
+
+#include "catalog/abstract_catalog.h"
+
+#define CONFIG_CATALOG_NAME "pg_settings"
+
+namespace peloton {
+namespace catalog {
+
+class ConfigCatalog : public AbstractCatalog {
```

Comment:
What are we calling this `ConfigCatalog`? It should be called `SettingsCatalog` to match the naming convention of the other catalog objects.